transfer/rsync/server.go

```diff
@@ -254,6 +303,12 @@ func NewServer(ctx context.Context, c client.Client, logger logr.Logger,
 		namespace = pvcList.Namespaces()[0]
 	}
 
+	if podOptions.SCCName == nil || (podOptions.SCCName != nil && *podOptions.SCCName == "") {
+		//TODO: raise a warning event
```

Comment:
I'm not sure whether this would be a warning or not... If you're not on OpenShift, lack of an SCC name doesn't matter.